cloud/endagaweb/templates/dashboard/index.html

```diff
@@ -23,6 +23,54 @@
     display: none;
   }
 </style>
+<html lang="en">
+<head>
+  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+  <script>
+
+$(document).ready(function(){
+$("#dialog").modal('show');
+});
+
+ </script>
+</head>
+<body>
+{% if messages %}
+<div class='modal fade' id='dialog'>
+  <div class='modal-dialog'>
+    <div class='modal-content'>
+      <div class='modal-header'>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class='modal-title'>
+          Password Expiry Status
+        </h4>
+      </div>
+      <div class='modal-body'>
+        {% for message in messages %}
```

Comment:
thanks,yes, there is one message need to prompt.But as per django message framework, message it store in FallbackStorage is the default storage.As per its definition This class first uses CookieStorage for all messages, falling back to using SessionStorage for the messages that could not fit in a single cookie. Since it is uses SessionStorage, it also requires Django's contrib.sessions application.
so it seems need to iterate in for loop for displaying  message in template.If we not iterate and directly shows messages, it show like
Hi <django.contrib.messages.storage.fallback. at 0x7f1b7cb75590> Please change your password.